docs/arch/adr-xx.md

```diff
@@ -0,0 +1,65 @@
+  - Some of the drawing tools use _LHC_
+- Some tools' annotation use `angle` some use `rotation`
+- It's unclear when the x, y annotation refers to the center point of the shape
+- It's unclear when the x, y annotation is being used as the point of rotation
+
+Some of the mark annotation models have a few other issues as well:
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` or `[0, 360)` for consistent angle calculation
+- The annotation model should use `angle` for naming
+- The annotation model should consistency use `x_center` and `y_center` for shapes where that is applicable
+  - The exceptions are point, line, fan. The fan tool is not symmetric. 
+  - Shape clustering in aggregation is always done with the center
+- All rotations should be defined about `x_center` and `y_center` point. If the rotation cannot be defined around the center point, then the point used that should be clearly recorded in the annotation as `x_rotation` and `y_rotation`
+- Default values should be removed wherever possible. This may result in some tools drawing UX changing. 
+  - The tools that have defaults are ellipse, rotate rectangle, fan. 
+  - A proposed example for the ellipse may involve requiring the volunters to click twice to set the the two radii. ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tools mark annotation will be a string of the SVG's path and it will be the responsibility of post-classification analysis to convert this to usable x,y points. We will include a sample script in the `DataDigging` repo for project owners to reference on how to do this. Aggregation in Caesar will have to be updated to do the conversion first.
+- An added `toolType` string name definition will be included. This would be `point`, `ellipse`, etc. This enables the aggregation for Caesar code to auto-configure which extractor to use without checking data types ([comment](https://github.com/zooniverse/front-end-monorepo/issues/823#issuecomment-493896524)).
+  - `tool` will change to `toolIndex` to clarify it is referring to the index of the input from the task area.
+  - The parent drawing task will have a `taskType` added as well.
```

Comment:
Just a point of clarification here, tasks and drawing tools both have `type` on their models, so I'm proposing a more explicit naming here. This may be unnecessary perhaps. I'm open to discussion on it.

Reply:
How would this differ from the existing types?

Reply:
I'm just proposing a rename of the existing `type` on both the task model and the tool model, though I didn't write that here clearly, but thinking about this again I've decided its unnecessary, so I'm going to remove this.